src/odemis/driver/test/orsay_test.py

```diff
@@ -46,6 +46,10 @@
 CONFIG_FIBVACUUM = {"name": "fib-vacuum", "role": "fib-vacuum"}
 CONFIG_FIBSOURCE = {"name": "fib-source", "role": "fib-source"}
 CONFIG_FIBBEAM = {"name": "fib-beam", "role": "fib-beam"}
+CONFIG_LIGHT = {"name": "light", "role": "light"}
+CONFIG_SCANNER = {"name": "scanner", "role": "scanner"}
+CONFIG_FOCUS = {"name": "focus", "role": "focus", "rng": (-2e-3, 2e-3),
+                "metadata": {model.MD_CALIB: 0.18e6}}  # MD_CALIB is in [V/m]
```

Comment:
No, change to a call to `updateMetadata`.

Reply:
I don't fully understand how the metadata should be handled then. Should or shouldn't they be specified in the yaml file? If not, then where should the 0.18e6 be defined? And how should it end up in the metadata?

Reply:
The metadata will/should be in the microscope file. That's actually what you've done in the example microscope file.
When the backend reads the microscope file and instantiate it, it does two things (that matters for here). First, it instantiate the component, with the "init" as argument. Then, it calls `comp.updateMetadata()` with the metadata from the microscope file.
So in the test case, you should simulate the same behaviour: instantiate the component (without the "metadata" argument), and immediately after call `updateMetadata({model.MD_CALIB: 0.18e6})`.

Reply:
For the case that the user hasn't put some metadata in the yaml file, it's good that the component still behaves not completely weird. So you could start with an internal default value for model.MD_CALIB as `0.2e6` or `0.1e6` to provide a rough value.